aws/graph-explorer
# Fix PNPM mono repo setup

## Description

Moves all repo related activities to the root folder.

Before, you had to open VS Code in the `/packages/graph-explorer` folder, or the `/packages/graph-explorer-proxy-server` folder. If you needed both you just had to jump around in different windows. If you tried to open the root folder in VS Code then Prettier & ESLint would not work. Also, managing dependencies was confusing because there were PNPM lock files in the root and each package folder.

## Validation



- Open VS Code at the root and ensure Prettier and ESLint work
- Run `pnpm start` for development
- Read through documentation changes
- Open a test file and run Jest Current File task
- Run `docker build .` to ensure Docker build succeeds

## Related Issues

Addresses #303 

### Check List
